src/lsst/cmservice/web_app/static/js/script-grid-utils.js

```diff
@@ -0,0 +1,192 @@
+// list of statuses to reset to if current status is failed or rejected
+const resetFromRejected = {
+    WAITING: 0,
+    READY: 1,
+    PREPARED: 2
+};
+// list of statuses to reset to if current status is reviewable
+const resetFromReviewable = {
+    ACCEPTED: 5,
+    REJECTED: -3,
+};
+
+const updateStatus = () => {
+    const rowIndex = parseInt(document.querySelector("#rowIndex").innerText);
+    const newStatus = parseInt(document.querySelector("#targetStatus").value);
+    const fullname = String(document.querySelector("#scriptFullname").innerText)
+    const scriptId = scriptGridApi.getRowNode(rowIndex).data.id;
+
+    let url;
+    let resetJson = null;
+
+    // accept script
+    if(newStatus === 5){
```

Comment:
I think you can do `newStatus == resetFromReviewable.ACCEPTED` instead of using a magic number here. It should improve how readable/ maintainable the code is.

Reply:
Sure, will do